fawltydeps/main.py

```diff
@@ -88,7 +88,7 @@ def success_message(check_undeclared: bool, check_unused: bool) -> Optional[str]
         return None
 
     @classmethod
-    def create(cls, settings: Settings) -> "Analysis":
+    def create(cls, settings: Settings, pull_deps: bool = False) -> "Analysis":
```

Comment:
Because I didn't want to deal with that yet at the time :sweat_smile: 
Added in [1443b05](https://github.com/tweag/FawltyDeps/pull/252/commits/1443b05ccd31adca20290f8069b3315d70281668).